framework/source/class/qx/ui/form/Slider.js

```diff
@@ -59,6 +59,7 @@ qx.Class.define("qx.ui.form.Slider",
   extend : qx.ui.core.Widget,
   implement : [
     qx.ui.form.IForm,
+    qx.ui.form.IField,
     qx.ui.form.INumberForm,
```

Comment:
Just curious: why is @qx.ui.fomr.IField@ needed here? As @qx.ui.form.INumberForm@ derives from @qx.ui.form.IField@ adding @qx.ui.form.INumberForm@ should be enough. Or do I misunderstand something here?